.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Definitely remove this file from src code control.  Please add `.vscode` to `.gitignore` while you're at it.